modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
++
+----
+rtfctl apply secure-property --key <key> --value <value> -n <environment>
+----
+
+* Applies a system configuration on a host node
++
+----
+rtfctl apply system-configuration
+----
+
+* Applies a system configuration on a host node without downloading the latest scripts
++
+----
+rtfctl apply system-configuration --skip-download
+----
+
+* Applies a Mule license
++
+----
+rtfctl apply mule-license '<BASE64-encoded-license>'
+----
+* Applies a monitoring proxy
++
+----
+rtfctl apply monitoring-proxy "socks5://<username>:<password>@<monitoring-URL>:<PORT>"`
+----
+
+* Clears a monitoring proxy configuration
++
+----
+rtfctl apply monitoring-proxy ""
+----
+
+=== `update` 
+
+Updates `rtfctl`.
+
+* Updates from the US control plane
++
+----
+rtfctl update
+----
+
+* Updates `rtfctl` from the EU control plane
++
+----
+rtfctl update --host eu1.anypoint.mulesoft.com
+----
+
+=== `wait`
+Waits for Runtime Fabric components to become ready
+
+* Specifies the number of seconds to wait 
+----
+rtfctl wait <value in seconds>
+----
+
+* Specifies the Number of seconds to pass before timing out the wait (Default is 600 seconds)
+
+----
+rtfctl wait --timeout <value in seconds>
+----
+
+== Commands for Installing and Upgrading Runtime Fabric
+
+Use the following commands when installing or upgrading Runtime Fabric, or when performing a back up or restore. 
 
 [%header%autowidth.spread,cols="a,a"]
-.rtfctl Flags
 |===
-|Flag |Description
-| -h, --help | Displays `rtfctl` help information.
-| -n, --namespace string | If present, displays the namespace scope.
-| --pod string | If present, displays the pod name.
+|Command |Description
+|xref:install[`install`] | Installs Runtime Fabric
+|xref:reinstall[`reinstall`] | Re-installs Runtime Fabric components
+|xref:upgrade-cluster[`upgrade-cluster`] | Upgrades the Runtime Fabric cluster
+|xref:backup[`backup`] | Performs a back up of the local Runtime Fabric state
+|xref:restore[`restore`] | Restores local Runtime Fabric state from a backup
+|xref:validate[`validate`] | Validates the cluster for Runtime Fabric installation
+|xref:version[`version`] | Displays the Runtime Fabric version information
 |===
 
-== Usage
+=== `install`
+Installs Runtime Fabric
 
-* To list all supported commands, run `rtfctl -h`. 
+----
+rtfctl install <activation data>
+----
 
-* For more information about a specific command, run `rtfctl <command> -h`.
+=== `reinstall`
+
+Re-installs Runtime Fabric components
+
+* Re-installs Runtime Fabric components using the latest scripts.
++
+----
+rtfctl reinstall
+----
+
+* Re-installs Runtime Fabric components without downloading the latest scripts.
++
+----
+rtfctl reinstall --skip-download
+----
+
+=== `backup`
+Performs a back up of the local Runtime Fabric state. 
+----
+rtfctl backup /var/backups/<backup-filename>.tar.gz
+----
+
+=== `restore`
+Restores local Runtime Fabric state from a backup 
+
+----
+rtfctl restore /var/backups/<filename>.tar.gz
+----
+
+=== `upgrade-cluster` 
```

Comment:
We can remove this section and any references to it. Its an hidden, deprecated command